### DISCUSSION TITLE
Computing SINR at a given position from Coverage Maps?

### DISCUSSION THREAD

**Discussion by anindha-dhar** [NVlabs/sionna#424] [Q&A]
I have multiple transmitters in the scene. I can generate coverage map. But my question is how can I compute SINR at a given position from Coverage Maps and then estimate the resulting BER from it? 

I have also multiple receivers and they also have mobility. I want to handover the transmission from one transmitter to another transmitter if the BER crosses a threshold_BER value. How can I achieve this? Can we use any new features from the latest version of sionna?

Thanks in advance.

**Comment by faycalaa** [comment]
Hi,

If multiple transmitters are defined, a coverage map is computed for every transmitter by Sionna. From the transmitters' coverage maps, you can compute an SINR for any transmitter $n$ for a given receiver position, considering transmitter  $n$ as the signal source and the others as interferers. Note that the coverage map indicates the path loss, not the received power. To calculate the received power, you must scale the path loss by the transmit power for every transmitter. 

Concerning your second question, you can define multiple transmitters and receivers, and simulate mobility of the receivers by iteratively updating their [position](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Receiver.position), [re-computing the coverage maps](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Scene.coverage_map), and then the BERs.

**Reply by anindha-dhar** [reply]
How can we now compute SINR at a given position using the new version of sionna? I think I can generate the coverage map and use 'SINR" metric. But if I want to know the SINR at a particular position what should I do?

**Reply by jhoydis** [reply]
Hi @anindha-dhar,

What do you mean by a "particular position", a particular cell of the coverage map or a particular position defined by its (x,y,z) coordinates? In the latter case, you can simply compute the channel impulse response for the desired and all interfering transmitters and compute the SINR manually from there.
